src/main/java/jenkins/plugins/slack/workflow/SlackSendStep.java

```diff
@@ -286,10 +298,11 @@ protected SlackResponse run() throws Exception {
                 return null;
             }
 
-            SlackService slackService = getSlackService(
-                    baseUrl, teamDomain, botUser, channel, step.replyBroadcast, iconEmoji, username, populatedToken);
+            SlackService slackService = getSlackService(baseUrl, teamDomain, botUser, channel, step.replyBroadcast, sendAsText, iconEmoji, username, populatedToken);
             final boolean publishSuccess;
-            if (step.attachments != null) {
+            if (sendAsText) {
```

Comment:
@casz sendAsText is used here to skip over creating the message as an attachment